RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 * [*] [internal] My Site Dashboard: Made some changes to the code architecture of the dashboard. The majority of the changes are related to the posts cards. It should have no visible changes but could cause regressions. Please test it by creating/trashing drafts and scheduled posts and testing that they appear correctly on the dashboard. [#18405]
 * [*] Quick Start: Updated the Stats tour. The tour can now be accessed from either the dashboard or the menu tab. [#18413]
 * [*] Quick Start: Updated the Reader tour. The tour now highlights the Discover tab and guides users to follow topics via the Settings screen. [#18450]
+* [*] Quick Start: Deleted the Edit your homepage tour. [#18469]
```

Comment:
I think this one deserves an `internal`. Deleting a tour is not relevant to be announced.

Reply:
Fixed! 0a03fbf